bumblebee/modules/datetime.py

```diff
@@ -30,6 +30,9 @@ class Module(bumblebee.engine.Module):
     def __init__(self, engine, config):
         super(Module, self).__init__(engine, config,
             bumblebee.output.Widget(full_text=self.get_time))
+        self._app = self.parameter("open", "calendar")
```

Comment:
All modules automatically have a parameter called "left-click" for binding against executables. My suggestion is to keep "calendar" as the default (by registering the callback like below), but to not make the application itself configurable (i.e. please remove the parameter "open"). Otherwise, there'd be two ways to achieve the same thing (datetime.open and datetime.left-click).

Reply:
Got it, I'll change that :)

Reply:
Actually I do not quite understand why you don't want to have the app configurable. I'd like to use google calendar as click action and it's rather ugly to edit the python files instead using a configuration value.

Reply:
Yes, "left-click" *should* be available for each module. It might be a bit tricky with the "datetime" module, as it has aliases. So, the following should work:

`bumblebee-status -m date -p date.left-click=calendar`
`bumblebee-status -m datetime -p datetime.left-click=calendar`
`bumblebee-status -m time -p time.left-click=calendar`

If it doesn't please let me know, 'cause that is a bug :)